client/components/theme/more-button.jsx

```diff
@@ -54,7 +54,11 @@ class ThemeMoreButton extends Component {
 
 		return (
 			<span className={ classes }>
-				<button ref={ this.moreButtonRef } onClick={ this.togglePopover }>
+				<button
+					aria-label={ `More options for theme ${ this.props.themeName }` }
```

Comment:
Since we're adding a prop to this component, we should also update the prop types at the bottom of the file. https://github.com/Automattic/wp-calypso/blob/3b91ba90f239db8c3d360e92575ee6c037446409/client/components/theme/more-button.jsx#L109

Reply:
This should be resolved with the newest commit.